CorsixTH/Lua/room.lua

```diff
@@ -493,19 +493,37 @@ function Room:commandEnteringStaff(humanoid, already_initialized)
   end
 end
 
+
+
```

Comment:
Don't think these two newlines are required? But it's not blocking if we left them in imo.